bootstrap.sh

```diff
@@ -23,24 +24,33 @@ last_bootstrapping_line_number=$(grep --line-number '^## Writing the Legislation
 
 cd ..
 pwd
-mv country-template openfisca-$lowercase_country_name
+mv country-template-master openfisca-$lowercase_country_name
 cd openfisca-$lowercase_country_name
 
 git init
 git add .
 git commit --message 'Initial import from OpenFisca country-template' --author='OpenFisca Bot <bot@openfisca.org>'
```

Comment:
This is extremely painful if you're [signing your work](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work) by default.

Reply:
Can you please elaborate on why this is painful? 🙂 I'm also signing and don't have a problem.
If the pain is that a password is requested twice in the middle of the script, I don't think there's much we can do to alleviate this choice. I would expect users that sign by default to use a `gpg-agent`.

Reply:
Hi @MattiSG, it is not because of password, but because commit fails unless you:

- have a global user setup
- add your user config before commiting

I added this as a comment as it is not a blocking issue for me.

Reply:
Actually adding `--no-gpg-sign` solves the issue 😃